src/cosmology/cosmology_functions.cpp

```diff
@@ -20,29 +24,69 @@ void Grid3D::Initialize_Cosmology(struct Parameters *P)
   chprintf("Cosmology Successfully Initialized. \n\n");
 }
 
+Real Cosmology::dtda_cosmo(Real da, Real a)
+{
+  // Return dt/da * da
```

Comment:
Minor formatting suggesting in case we ever want to generate doxygen documentation

```suggestion
/* Computes dt/da * da */
Real Cosmology::dtda_cosmo(Real da, Real a)
{
```

Reply:
I'll default to this comment format in the future.